File: app/src/main/java/fr/scootop/data/model/configuration/Poste.java
package fr.scootop.data.model.configuration;
import java.util.List;

import fr.scootop.data.model.user.domain.Player;


public class Poste {
	
	private Long id;
	
	private String name;
	
	@JsonIgnoreProperties({"postes","statisticalSheets"})
	private List<Player> players;
}

File: app/src/main/java/fr/scootop/data/model/configuration/Poste.kt
package fr.scootop.data.model.configuration

import android.os.Parcel
import android.os.Parcelable
import fr.scootop.data.model.user.domain.Player

class Poste() : Parcelable {
    val id: Long? = null
    val name: String = ""
    val players: List<Player>? = null

    constructor(parcel: Parcel) : this() {
    }

    override fun toString(): String {
        return name
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {

    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Poste> {
        override fun createFromParcel(parcel: Parcel): Poste {
            return Poste(parcel)
        }

        override fun newArray(size: Int): Array<Poste?> {
            return arrayOfNulls(size)
        }
    }


}